lighthouse-core/audits/byte-efficiency/unused-javascript.js

```diff
@@ -15,7 +15,7 @@ const UIStrings = {
   title: 'Remove unused JavaScript',
   /** Description of a Lighthouse audit that tells the user *why* they should remove JavaScript that is never needed/evaluated by the browser. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Remove unused JavaScript to reduce bytes consumed by network activity. ' +
-    '[Learn more](https://web.dev/remove-unused-code/).',
+    '[Learn more](https://web.dev/unused-javascript/).',
```

Comment:
* Staged: https://deploy-preview-3266--web-dev-staging.netlify.app/unused-javascript/
* PR: https://github.com/GoogleChrome/web.dev/pull/3266

Reply:
Aside: I'm excited to be able to link to Tooling.Report! Should make some of these audits much more actionable (and aligned with real-world DX).